tests/integration/redis/test_list.php

```diff
@@ -10,9 +10,9 @@
 
 /*SKIPIF
 <?php
-if (version_compare(phpversion(), '8.0.0', '>=')) {
-    die("skip: PHP < 8.0.0 required\n");
-} 
+if (version_compare(phpversion(), '5.4', '<')) {
```

Comment:
Does this test work with PHP >= 8.0.0 now?

Reply:
I think they likely can be removed, as I am calling into these deprecated PhpRedis tests in the non-version-specific test.

The only question is whether there is utility in explicitly checking for the "Deprecated" warning, which is being done in the `.php8` test.

Reply:
I'd vote for removing `tests/integration/redis/test_list.php8.php` and `tests/integration/redis/test_instance_list.php8.php`. The `tests/integration/redis/test_list.php` and `tests/integration/redis/test_instance_list.php` test the same methods. The only difference is that when 'Deprecation' is detected, `Errors` metrics are expected. Moreover it's not PHP 8 that deprecated `lGet` and `lRemove` redis methods - they were deprecated in redis/PhpRedis 5.0.0 so the test name is misleading.

Reply:
Resolved via 8571000